### PR TITLE
Feature/version desktop

### PR DESCRIPTION
**Pull request by Kgomotso196**
# ON THIS PROJECT I COULD:

1. Apply forms in HTML.
2. Learn the use of form spree.
3. Use flexbox for CSS layout.
4. Pair program.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -268,16 +266,45 @@ <h3 class="language-container-text">Skills</h3>
...
+            type="text"
+            class="ftext"
+            placeholder="Full name"
+            maxlength="30"
+            required
+          />
+          <input
+            type="email"
+            class="femail"
+            placeholder="Email address"
```
Comment (Shedrack-Sunday) [inline index.html:285]: - [ ] Good job so far on the project 👏 . However, your form area seems to be a little bit clustered from the original design given. this most definitely has to do with your `grid` and `flexbox` alignments, which I believe you can fix. 
- Kindly also align the background images accordingly 
- To make it better explained, I've attached comparison images with that of the Figma design to help you access them.
- These changes are needed to make your work have a better user view and look professional 👍 

## Your view :
image

## Figma:
image

File: index.html [hunk 2]
```diff
@@ -268,16 +266,45 @@ <h3 class="language-container-text">Skills</h3>
...
             <a href="#"><img src="svgFiles/twitterIcon.svg" alt="Twitter" /></a>
           </li>
           <li>
-            <a href="#"
-              ><img src="svgFiles/linkedinIcon.svg" alt="LinkedIn"
-            /></a>
+            <a href="#"><img src="svgFiles/linkedinIcon.svg" alt="LinkedIn"/></a>
           </li>
           <li>
             <a href="#"><img src="svgFiles/mediumIcon.svg" alt="Medium" /></a>
```
Comment (Shedrack-Sunday) [inline index.html:310]: - [ ] Kindly also adjust  the social icons accordingly. they should be fixed at the footer as given on the template. 👍 

## View:
image

**Review by Shedrack-Sunday — CHANGES_REQUESTED** [review]
Hi @Kgomotso196    👋 

# CHANGES REQUIRED ♻️

Great work on the project 👏
You've done well implementing the requirements,  However, there are a few you need to work on, to move to the next project. 💯

 



Highlights
✅All linters are checked
✅ Proper PR summary 
  
# Required Changes ♻️

- [x] I must admit you have really put in the effort so far in this project, however, the changes required have to do with a bit of styling.

_check the comment section for a detailed review_ 

# Optional suggestions 

Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application.

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.

As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Comment by Kgomotso196** [comment]
Hey there @Shedrack-Sunday I don't think I understand you, this is how it looks on my side.

Screenshot (146)

**Comment by Shedrack-Sunday** [comment]
> Hey there @Shedrack-Sunday I don't think I understand you, this is how it looks on my side.
> 
> Screenshot (146)

Hello @Kgomotso196, I did run the project locally and viewed it via live server. If it's so from your end and you're certain, you can request for second opinion or send it on the dashboard so another reviewer will take  a look.

Best Regards

File: otherCSSfiles/programmingLanguagesSection.css [hunk 3]
```diff
@@ -43,17 +60,67 @@
     display: flex;
     flex-wrap: wrap;
     justify-content: center;
+    list-style: none;
+    gap: 8px;
   }
   
-  @media screen and (min-width: 768px) {
+  @media screen and (max-width: 1180px) and (min-width: 768px) {
```
Comment (GedeonTS) [inline otherCSSfiles/programmingLanguagesSection.css:67]: - [ ] As requested by the previous reviewer kindly fix your design as it is still breaking 
 - You can use a single media query (like min-width:768px) to get more control on the wider width as I think using more media queries can hard to control
127 0 0 1_5502_index html

**Review by GedeonTS — CHANGES_REQUESTED** [review]
# STATUS: CHANGES REQUIRED :recycle: :
Hi @Kgomotso196 , 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.



 ## Highlights :tophat: :+1: 
- No linter errors :+1: 
 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions :ringed_planet: 

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: otherCSSfiles/programmingLanguagesSection.css (hunk 3 above)
Comment (Kgomotso196) [reply otherCSSfiles/programmingLanguagesSection.css:67]: Hello @GedeonTS I really understand what you guys are talking about. But my struggle is how do I make it appear as it does to you on my side, like so I can know when I remove the @media query I know how my project got changed?

**Review by DuaneDave — APPROVED** [review]
Hi @Kgomotso196 👋 , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
✔️ Your design looks good
✔️ PR summary present 
✔️ Correct usage of github flow


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**